test/e2e/utils/datainserter.go

```diff
@@ -157,8 +159,6 @@ func (di *DataInserter) AwaitSchemaAgreement(ctx context.Context) error {
 }
 
 func (di *DataInserter) Read() ([]*TestData, error) {
-	framework.Infof("Reading data from table %s", di.table.Name())
```

Comment:
Why delete it if Read() is not used in continuous reading?

Reply:
leftover from refactoring, brough it back, thanks